commands_test.go

```diff
@@ -0,0 +1,114 @@
+package main
+
+import (
+	"os/exec"
+	"testing"
+
+	drivers "github.com/docker/machine/drivers"
+	"github.com/docker/machine/state"
+)
+
+type FakeDriver struct {
+	MockState state.State
+}
+
+func (d *FakeDriver) DriverName() string {
+	return "fakedriver"
+}
+
+func (d *FakeDriver) SetConfigFromFlags(flags drivers.DriverOptions) error {
+	return nil
+}
+
+func (d *FakeDriver) GetURL() (string, error) {
+	return "", nil
+}
+
+func (d *FakeDriver) GetIP() (string, error) {
+	return "", nil
+}
+
+func (d *FakeDriver) GetState() (state.State, error) {
+	return d.MockState, nil
+}
+
+func (d *FakeDriver) Create() error {
+	return nil
+}
+
+func (d *FakeDriver) Remove() error {
+	return nil
+}
+
+func (d *FakeDriver) Start() error {
+	return nil
+}
+
+func (d *FakeDriver) Stop() error {
+	return nil
+}
+
+func (d *FakeDriver) Restart() error {
+	return nil
+}
+
+func (d *FakeDriver) Kill() error {
+	return nil
+}
+
+func (d *FakeDriver) Upgrade() error {
+	return nil
+}
+
+func (d *FakeDriver) GetSSHCommand(args ...string) (*exec.Cmd, error) {
+	return &exec.Cmd{}, nil
+}
+
+func TestGetHostState(t *testing.T) {
+	storePath := "~/.docker"
```

Comment:
What do you mean smash current configs?  What else would we change it to?  I mostly just have it here as a dummy value so it can fill in the field.